src/Voron/Data/Tables/TableSchema.IndexDefinitions.cs

```diff
@@ -362,8 +376,25 @@ public static DynamicKeyIndexDef ReadFrom(ByteStringContext context, ref TableVa
                 // read if this index has OnEntryChanged
                 currentPtr = input.Read(5, out currentSize);
                 var hasIndexEntryChangedDelegate = Convert.ToBoolean(*currentPtr);
-                if (hasIndexEntryChangedDelegate == false) 
-                    return indexDef;
+                if (hasIndexEntryChangedDelegate == false)
+                {
+                    if (input.Count == 6)
+                    {
+                        indexDef.SupportDuplicateKeys = false;
+                        return indexDef;
+                    }
+
+                    if (input.Count == 7)
+                    {
+                        // read if this index has SupportDuplicateKeys
+                        currentPtr = input.Read(6, out currentSize);
```

Comment:
I see that in this file have hard coded indexes to read from. Like `6` here. 

While in other places we use enums so we have:

```
tvr.Read((int)CountersTable.CounterKey, out var size);
```
or
```
LowerId = TableValueToString(context, (int)DocumentsTable.LowerId, ref tvr),
```
This is really helpful when reading or debugging the code. Can we have something like that here too?

Reply:
done